src/Microsoft.Health.Dicom.Core/Features/Telemetry/InstanceMeter.cs

```diff
@@ -0,0 +1,43 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Diagnostics.Metrics;
+
+namespace Microsoft.Health.Dicom.Core.Features.Telemetry;
+
+public sealed class InstanceMeter : IDisposable
```

Comment:
Seeing the `StoreMeter` got me thinking. Maybe the meters should be aligned with the DICOMweb services. Perhaps there should really be an `StoreMeter`, `QueryMeter`, RetrievalMeter` (`RetrieveMeter` ?), etc (or alernatively `StowMeter`, QidoMeter`, and `WadoMeter`, etc). Then for the API controllers we made up would perhaps be like the `DeletionMeter`, `OperationMeter`, etc). Do you think this makes sense?

For backend-specific meters, perhaps they have their own nested meters depending on the operation.

Eg. `StoreMeter` is called `Microsoft.Health.Dicom.Store` while for blob-specific metrics there is a `BlobStoreMeter` which has the name `Microsoft.Health.Dicom.Store.Blob`.

Reply:
Yea, we can do it in different ways. Josiah had suggested to keep the metrics granulated at project level. I believe we can keep the current meters as such for now and refactor if required if and when we add more instruments.

Reply:
Synced offline and named the meters as `StoreMeter`, `RetrieveMeter` and `DeleteMeter`